app/models/agents/data_output_agent.rb

```diff
@@ -156,6 +157,12 @@ def feed_description
       interpolated['template']['description'].presence || "A feed of Events received by the '#{name}' Huginn Agent"
     end
 
+    def xml_namespace
+      if (interpolated['itunes_compatible'] == "true")
+      	"xmlns:itunes=\"http://www.itunes.com/dtds/podcast-1.0.dtd\""
```

Comment:
OK. It'd probably be a good idea to rename the option from `itunes_compatible` to something like `ns_itunes`, to leave it consistent for adding other optional namespaces in the future